bikespace_frontend/src/components/landing/landing-layout/landing-layout.module.scss

```diff
@@ -10,6 +10,8 @@ $even-lighter-grey: #EEE;
   margin: 0 auto;
   padding: 1rem;
   max-width: $wrapper-max-width;
+  display: flex;
```

Comment:
I would try and avoid doing this and control the justification of content on a block by block basis instead of at the page level.

Reply:
Updated